vagrant.gemspec

```diff
@@ -16,6 +16,8 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = ">= 1.3.6"
 
   s.add_dependency "base64", "~> 0.2.0"
+  s.add_dependency "logger", "~> 1.6.1"
```

Comment:
This line should probably be changed to `~>1.0` or `~>1.6`, refer to #13527 for more info.

Reply:
Or say ~> 1.6.0?

Reply:
I don't speak Ruby so I'm not sure (shrugs), I'll leave the judgement to you.